src/main/resources/application.yaml

```diff
@@ -337,6 +340,9 @@ darts:
       unstructured-to-arm-batch-processor:
         max-arm-manifest-items: 100
         max-arm-single-mode-items: 100
+      dets-to-arm-batch-processor:
+        max-arm-manifest-items: 100
```

Comment:
is this 100 not common across the whole application?

Reply:
This can be changed per automated task and is specific to arm so its how many items they can accept in batch mode.